pig/run_pig.sh

```diff
@@ -275,4 +275,5 @@ else
 	# Save the results for later.
 	#
 	tar hcf /tmp/results_${test_name}_${to_tuned_setting}.tar results_${test_name}_${to_tuned_setting}
+	${curdir}/test_tools/save_results --curdir $curdir --home_root $to_home_root --tar_file /tmp/results_${test_name}_${to_tuned_setting} --test_name $test_name --tuned_setting=$to_tuned_setting --version NONE --user $to_user
```

Comment:
In previous PR's, I believe you were deleting the previous tar command.  Is there a reason why it's staying in this one?

See https://github.com/redhat-performance/uperf-wrapper/pull/27/files for reference